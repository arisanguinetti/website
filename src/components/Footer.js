import React, { Component } from "react";
import moment from "moment";

class Footer extends Component {
  render() {
    const currentYear = moment().year();
    return (
      <footer className="footer">
        <div className="container">
          <div className="row">
            <div className="col-md-12">
              <div className="page-scroll">
                <a href="#page-top">
                  <i className="icon-Arrow-Up" />
                </a>
              </div>

              <ul className="list-inline social-icons">
                <li>
                  <div className="item">
                    <a
                      href="https://www.facebook.com/arisanguinetti"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      <i className="fa fa-facebook" />
                    </a>
                  </div>
                </li>

                <li>
                  <div className="item">
                    <a
                      href="https://twitter.com/arisanguinetti"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      <i className="fa fa-twitter" />
                    </a>
                  </div>
                </li>

                <li>
                  <div className="item">
                    <a
                      href="https://www.linkedin.com/in/arisanguinetti"
                      target="_blank"
                      rel="noopener noreferrer"
                    >
                      <i className="fa fa-linkedin" />
                    </a>
                  </div>
                </li>
              </ul>
            </div>
          </div>

          <div className="row">
            <div className="col-md-12">
              <div className="copyright text-center">
                <p className="text-grey">
                  &copy; {currentYear}. All rights reserved.
                </p>
              </div>
            </div>
          </div>
        </div>
      </footer>
    );
  }
}

export default Footer;
