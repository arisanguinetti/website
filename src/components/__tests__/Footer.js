import React from "react";
import { shallow } from "enzyme";
import Footer from "../Footer";

describe("Footer", () => {
  describe("Rendering", () => {
    it("should match to snapshot", () => {
      const component = shallow(<Footer />);
      expect(component).toMatchSnapshot();
    });
  });
});
