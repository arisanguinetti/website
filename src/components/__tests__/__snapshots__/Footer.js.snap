// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Rendering should match to snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="page-scroll"
        >
          <a
            href="#page-top"
          >
            <i
              className="icon-Arrow-Up"
            />
          </a>
        </div>
        <ul
          className="list-inline social-icons"
        >
          <li>
            <div
              className="item"
            >
              <a
                href="https://www.facebook.com/arisanguinetti"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fa fa-facebook"
                />
              </a>
            </div>
          </li>
          <li>
            <div
              className="item"
            >
              <a
                href="https://twitter.com/arisanguinetti"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fa fa-twitter"
                />
              </a>
            </div>
          </li>
          <li>
            <div
              className="item"
            >
              <a
                href="https://www.linkedin.com/in/arisanguinetti"
                rel="noopener noreferrer"
                target="_blank"
              >
                <i
                  className="fa fa-linkedin"
                />
              </a>
            </div>
          </li>
        </ul>
      </div>
    </div>
    <div
      className="row"
    >
      <div
        className="col-md-12"
      >
        <div
          className="copyright text-center"
        >
          <p
            className="text-grey"
          >
            Â© 
            2019
            . All rights reserved.
          </p>
        </div>
      </div>
    </div>
  </div>
</footer>
`;
